discovery_engine_core/ai/bert/src/lib.rs

```diff
@@ -12,18 +12,30 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-#![cfg_attr(
-    doc,
-    forbid(rustdoc::broken_intra_doc_links, rustdoc::private_intra_doc_links)
-)]
-#![forbid(unsafe_op_in_unsafe_fn)]
-//! The RuBert pipeline computes embeddings of sequences.
+//! The Bert pipelines compute embeddings of sequences.
 //!
 //! Sequences are anything string-like and can also be single words or snippets. The embeddings are
 //! f32-arrays and their shape depends on the pooling strategy.
 //!
 //! See the example in this crate for usage details.
 
+#![forbid(unsafe_op_in_unsafe_fn)]
+#![deny(
+    clippy::pedantic,
+    clippy::future_not_send,
+    noop_method_call,
+    rust_2018_idioms,
+    unsafe_code,
```

Comment:
we can only `deny(unsafe_code)` but not `forbid(unsafe_code)` as usual because the `ndarray::s!` macro expands an `allow(unsafe_code)` when we call it.